include/modem/lte_lc.h

```diff
@@ -958,7 +961,7 @@ struct lte_lc_conn_eval_params {
  *
  * When using search types up to @ref LTE_LC_NEIGHBOR_SEARCH_TYPE_EXTENDED_COMPLETE, the result
  * contains parameters from current/serving cell and optionally up to
- * @kconfig{CONFIG_LTE_NEIGHBOR_CELLS_MAX} neighbor cells.
+ * `CONFIG_LTE_NEIGHBOR_CELLS_MAX` neighbor cells.
```

Comment:
@peknis, this kconfig syntax is used ~150 times in nRF so is there an actions somewhere to clean this if it's not working properly in the document generation.
And was this broken at some point as I'd guess it has been working better in the past...

Reply:
It's not working in Doxygen generated content. RSTs should still work.

Reply:
This might be worth a Jira ticket. Let's see. Here is some background for the change:
Why it (Breathe) was removed (from Gerard's comment in one of the JIRAs): breathe, the extension we used, had a poor performance (with removal CI went from ~40min to ~15min in NCS) and was not maintained properly. Upstream Zephyr proceeded with the removal of it and so NCS followed.

Reply:
Confirming that Kconfig links in rst still works as usual.

Reply:
Ok, nothing we can really do but one more thing downgrading our API reference documentation.

Reply:
Unfortunately yes.